steps/logger.py

```diff
@@ -1,21 +1,25 @@
 import logging
-def setup_logging(log_level, log_path, log_format_json):
+import os
+
+
+def setup_logging(log_path):
+    log_level = os.getenv(ADG_LOG_LEVEL)
```

Comment:
this will fail if env var is missing or not uppercase, so I suggest the following:
```suggestion
    log_level = os.environ["ADG_LOG_LEVEL"].upper() if "ADG_LOG_LEVEL" in os.environ else "INFO"
```